SharedContent/cpp-winui-template/WinUI3TemplateCpp/SettingsPage.xaml.cpp

```diff
@@ -0,0 +1,77 @@
+﻿#include "pch.h"
+#include "SettingsPage.xaml.h"
+#if __has_include("SettingsPage.g.cpp")
+#include "SettingsPage.g.cpp"
+#endif
+#include <SampleConfiguration.h>
+
+using namespace winrt;
+using namespace Microsoft::UI::Xaml;
+using namespace Microsoft::UI::Xaml::Controls;
+
+namespace winrt::WinUI3TemplateCpp::implementation
+{
+    SettingsPage::SettingsPage()
+    {
+        InitializeComponent();
+    }
+
+    void SettingsPage::OnNavigatedTo(Microsoft::UI::Xaml::Navigation::NavigationEventArgs const& e)
+    {
+        // Initialize the CurrentTheme on first navigation to the Settings page
+        if (winrt::WinUI3TemplateCpp::Settings::CurrentTheme == L"")
```

Comment:
Consistency - below you don't repeat the `winrt::WinUI3TemplateCpp` namespace on `Settings::X`; choose one